packages/salesforcedx-sobjects-faux-generator/test/fauxClassGenerator.test.ts

```diff
@@ -0,0 +1,293 @@
+import * as chai from 'chai';
+import * as fs from 'fs';
+import { FauxClassGenerator } from '../src/generator/fauxClassGenerator';
+
+const expect = chai.expect;
+
+describe('SObject faux class generator', function() {
+  let classPath = '';
+
+  afterEach(() => {
+    if (classPath) {
+      try {
+        fs.unlinkSync(classPath);
+      } catch (e) {
+        console.log(e);
+      }
+      classPath = '';
+    }
+  });
+
+  it('Should generate a faux class with all types of fields that can be in custom SObjects', async () => {
+    const fieldsHeader = '{ "name": "Custom__c", "fields": [ ';
+    const closeHeader = ' ], "childRelationships": [] }';
+
+    const fields: string[] = [
+      '{"name": "Foo1", "type": "string", "referenceTo": []}',
```

Comment:
Maybe we should call these like `StringType` instead of `Foo1`. It's more explicit what we are testing. It's a small change but I think it adds clarity. I do not like names like Foo1, Foo2, etc.

Reply:
will do